src/test/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformerTest.java

```diff
@@ -243,6 +243,15 @@ public void givenAnInvalidMrnDate_thenAddErrorToList() {
         assertTrue(result.getErrors().contains("mrn_date is an invalid date field. Needs to be in the format dd/mm/yyyy"));
     }
 
+    @Test
```

Comment:
Also may be worth adding a test case for empty ?

Reply:
Done